quint/src/repl.ts

```diff
@@ -123,7 +123,8 @@ class ReplState {
     // Introduce the __repl__ module to the compilation state so new expressions
     // are pushed to it.
     const state = newCompilationState()
-    return { ...state, modules: [{ name: '__repl__', defs: simulatorBuiltins(state), id: 0n }] }
+    const modules = [{ name: '__repl__', defs: simulatorBuiltins(state), id: 0n }]
+    return { ...state, modules: modules, originalModules: modules }
```

Comment:
Afaik, the only purpose of `__repl__` is to carry auxiliary definitions. Since we have `modules` and `originalModules` now, would not it be easier just to save the auxiliary REPL definitions as an array of definitions and inject it in the main module?

Reply:
Yes, I think we should do that! I mentioned it in the last quint meeting. Will do in a followup.